src/measure_theory/function/uniform_integrable.lean

```diff
@@ -181,9 +188,9 @@ end
 
 end egorov
 
-variables [second_countable_topology β] [measurable_space β] [borel_space β]
-  {f : ℕ → α → β} {g : α → β} {s : set α}
-
+variables [semilattice_sup ι] [nonempty ι] [encodable ι]
+  [second_countable_topology β] [measurable_space β] [borel_space β]
+  {f : ι → α → β} {g : α → β} {s : set α}
 
 /-- **Egorov's theorem**: If `f : ℕ → α → β` is a sequence of measurable functions that converges
```

Comment:
```suggestion
/-- **Egorov's theorem**: If `f : ι → α → β` is a sequence of measurable functions that converges
```
perhaps with a remark that usually `ι = ℕ`

Reply:
Right. I did not look at the docstrings at all. I'll change them wherever needed.